src/SessionSetup/ConnectToStadiaWidget.cpp

```diff
@@ -561,31 +422,44 @@ void ConnectToStadiaWidget::TrySelectRememberedInstance() {
   remembered_instance_id_ = std::nullopt;
 }
 
-void ConnectToStadiaWidget::showEvent(QShowEvent* event) {
-  QWidget::showEvent(event);
-  // It is important that the call to DetachRadioButton is done here and not during construction.
-  // For high dpi display settings in Windows (scaling) the the actual width and height of the radio
-  // button is not known during construction. Hence the call is done when the widget is shown, not
-  // when its constructed
-  DetachRadioButton();
-}
-
 bool ConnectToStadiaWidget::IsActive() const { return ui_->contentFrame->isEnabled(); }
 
-void ConnectToStadiaWidget::SetProject(const std::optional<orbit_ggp::Project>& project) {
-  if (selected_project_ == project) return;
+void ConnectToStadiaWidget::Connect() {
+  CHECK(selected_instance_.has_value());
 
-  QSettings settings;
-  if (!project.has_value()) {
-    settings.setValue(kSelectedProjectDisplayNameKey, QVariant());
-    settings.setValue(kSelectedProjectIdKey, QVariant());
-  } else {
-    settings.setValue(kSelectedProjectDisplayNameKey, project.value().display_name);
-    settings.setValue(kSelectedProjectIdKey, project.value().id);
+  auto account_result = GetAccountSync();
+  if (account_result.has_error()) {
+    emit ErrorOccurred(QString::fromStdString(account_result.error().message()));
+    return;
+  }
+
+  if (!account_result.value().email.contains(selected_instance_->owner)) {
```

Comment:
This probably wasn't your first choice, but is this really the best / only way to verify these match? Could it happen that `owner` just happens to be part of another accounts email and this check doesn't work as expected?

Reply:
There can be multiple accounts in ggp. `ggp auth list` returns a list of accounts, of which always only one is the active account (in json "default"). Currently only this one account is used, which makes sense, because the list of projects is determined by the permissions of this active account. These accounts contain the full email (username@domain.com). An instance as retrieved by the instance list has an owner, which only contains the first username part of the email (username@). Thats why I am doing this "username@domain.com".contains("username@"). 

Now that I am thinking about it. It could be that the owner is "suffix@" and the account is "prefix_suffix@domain.com". In which case this does not what is intended. I am going to change this to specifically check if owner is a prefix of email.

Reply:
done